docs/cli/gittuf_clone.md

```diff
@@ -9,8 +9,9 @@ gittuf clone [flags]
 ### Options
 
 ```
-  -b, --branch string   specify branch to check out
-  -h, --help            help for clone
+  -b, --branch string              specify branch to check out
+  -e, --expectedRootKeys strings   specify expected root keys in cloned repo
```

Comment:
Some name nitting to try and make it clearer what this flag does (happy to go back and forth on this)

```suggestion
  -k, --root-keys strings   path(s) of root keys to verify when cloning the repo.
```

Reply:
Sorry for the late reply, fixed.